applications/plugins/SofaPython/PythonScriptController.h

```diff
@@ -133,6 +135,7 @@ class SOFA_SOFAPYTHON_API PythonScriptController : public ScriptController
     PyObject *m_Func_onMouseWheel;
     PyObject *m_Func_onGUIEvent;
     PyObject *m_Func_onScriptEvent;
+    PyObject* m_Func_onEvent;
```

Comment:
for the record: this is for handling arbitrary events directly from python controllers: events are passed as an opaque pointer (through a capsule IIRC) and python code can access it using ctypes.

I just removed it from this PR.